src/Azure.Functions.Cli/Helpers/KuduLiteDeploymentHelpers.cs

```diff
@@ -34,8 +33,22 @@ public static async Task<DeployStatus> WaitForRemoteBuild(HttpClient client, Sit
 
             while (statusCode != DeployStatus.Success && statusCode != DeployStatus.Failed && statusCode != DeployStatus.Unknown)
             {
-                statusCode = await GetDeploymentStatusById(client, functionApp, id);
-                logLastUpdate = await DisplayDeploymentLog(client, functionApp, id, logLastUpdate);
+                try
+                {
+                    statusCode = await GetDeploymentStatusById(client, functionApp, id);
+                } catch (HttpRequestException)
+                {
+                    return DeployStatus.Unknown;
+                }
+
+                try
+                {
+                    logLastUpdate = await DisplayDeploymentLog(client, functionApp, id, logLastUpdate);
+                } catch (HttpRequestException)
```

Comment:
nit: formatting
```suggestion
                } 
                catch (HttpRequestException)
```

Also you can merge the 2 try blocks, right?